commands/http/client.go

```diff
@@ -101,46 +94,28 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 	}
 	httpReq.Header.Set(uaHeader, config.ApiVersion)
 
-	ec := make(chan error, 1)
-	rc := make(chan cmds.Response, 1)
-	dc := req.Context().Done()
+	httpReq.Cancel = req.Context().Done()
+	httpReq.Close = true
```

Comment:
But there is not explicit close at elsewhere in the code. Either kill it after one request, or set a gc (which remains to be implemented).